services/oc-pmrmrg/VERSION_INTEGRATION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
hmm this looks wrong:
it should be 1.0.0 unless you used some old cookiecutter?